mobile/library/common/jni/jni_helper.cc

```diff
@@ -9,19 +9,19 @@ JNIEnv* JniHelper::getEnv() { return env_; }
 
 jmethodID JniHelper::getMethodId(jclass clazz, const char* name, const char* signature) {
   jmethodID method_id = env_->GetMethodID(clazz, name, signature);
-  rethrowException();
+  handleException();
```

Comment:
I don't see this mentioned in the PR description. Did we decide to swallow exceptions instead of re-throwing them?

Reply:
Sorry. I was playing around with it but I didn't intend to commit the changes, but I guess I accidentally had this committed. Reverted the change.